package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@draftbit/re-font-awesome",
-  "version": "1.2.0",
+  "name": "@draftbit/rescript-font-awesome",
```

Comment:
A package name change isn't warranted here. `re` works for both `rescript` and `reason`, which I'm sure was by design 😄

Reply:
https://forum.rescript-lang.org/t/naming-standard-for-new-binding-projects/572/2

Reply:
I appreciate the points of view in that thread but it's just a few people's opinions, not an agreed upon standard. Having multiple packages with similar names is confusing and I don't see a clear motivation for it here.

Reply:
The only reason I suggested this is it has been pretty much decided on by the community, if you take a look at any recently published packages, you will see that they all are using rescript prefix. Totally up to you, but this is where I see the community going.

https://rescript-lang.org/packages

Reply:
Saw another mention about the community naming convention today when on the forum: https://forum.rescript-lang.org/t/work-in-progress-rescript-express/1261/6